src/Toggle/UseCase/ToggleRetriever.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Clearbooks\Labs\Toggle\UseCase;
+
+use Clearbooks\Labs\Db\Entity\Toggle;
+
+interface ToggleRetriever
+{
+    /**
+     * @param int $toggleId
+     * @return Toggle|null
```

Comment:
there might be someone who wanna use getToggleById.
Oh by the way toggleId is string.